[04] View & Modifiers/[01] Challenge 1/WeSplit/ContentView.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  ContentView.swift
+//  WeSplit
+//
+//  Created by Joe Pham on 2022-07-12.
+//
+
+import SwiftUI
+
+struct ContentView: View {
+    var body: some View {
+		WeSplitView(check: Check())
```

Comment:
It could also be possible to create an instance of `Check` inside WeSplitView instead?